proto-gen/openapi/api_v2.swagger.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "api_v2.proto",
+    "title": "api_v2/collector.proto",
```

Comment:
This is a little confusing. The generated swagger file has all definitions from `sampling.proto` as well, but not sure why lists title as `api_v2/collector.proto`